README.md

```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
 > cargo build
 ```
 
-or to enable GTK 3.10 examples as well:
+or to enable GTK 3.1x, 3.2x, all examples as well:
```

Comment:
What about "or to enable GTK 3.x depending on the version needed by the example:"